controllers/che/checluster_controller.go

```diff
@@ -96,6 +97,8 @@ func NewReconciler(
 	openShiftOAuthUser := openshiftoauth.NewOpenShiftOAuthUser()
 	reconcileManager.RegisterReconciler(openShiftOAuthUser)
 	reconcileManager.RegisterReconciler(openshiftoauth.NewOpenShiftOAuth(openShiftOAuthUser))
+	reconcileManager.RegisterReconciler(tls.NewCertificates())
```

Comment:
`NewCertificates` sounds like the function creates new certificates, but actuallt it is a reconciler. Please think about better name.